adagrams/game.py

```diff
@@ -1,11 +1,155 @@
+import random
+
+letter_pool_list = ['A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A','B','B', 'C', 'C', 'D', 'D', 'D', 'D', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'F', 'F', 'G', 'G', 'G', 'H', 'H', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'J', 'K', 'L', 'L', 'L', 'L', 'M', 'M', 'N', 'N', 'N', 'N', 'N', 'N', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'P', 'P', 'Q', 'R', 'R', 'R', 'R', 'R', 'R', 'S', 'S', 'S', 'S', 'T', 'T', 'T', 'T', 'T', 'T', 'U', 'U', 'U', 'U', 'V', 'V', 'W', 'W', 'X', 'Y', 'Y', 'Z']
+
+LETTER_POOL_VALUES = {
+    'A': 1, 
+    'B': 3, 
+    'C': 3, 
+    'D': 2, 
+    'E': 1, 
+    'F': 4, 
+    'G': 2, 
+    'H': 4, 
+    'I': 1, 
+    'J': 8, 
+    'K': 5, 
+    'L': 1, 
+    'M': 3, 
+    'N': 1, 
+    'O': 1, 
+    'P': 3, 
+    'Q': 10, 
+    'R': 1, 
+    'S': 1, 
+    'T': 1, 
+    'U': 1, 
+    'V': 4, 
+    'W': 4, 
+    'X': 8, 
+    'Y': 4, 
+    'Z': 10
+}
+
 def draw_letters():
-    pass
+    '''
+    This function takes no arguments and generates a random set of letters to display
+    on the screen by making a copy of the letter_pool_list
+    output: returns an array of 10 strings that are random letters
+
+    '''
+
+    letter_pool_list_copy = list(letter_pool_list)
+    one_list = []
+  
+    for i in range(10):
+        random_letter = random.choice(letter_pool_list_copy)
+        one_list.append(random_letter)
+        letter_pool_list_copy.remove(random_letter)
+    return one_list
+
 
 def uses_available_letters(word, letter_bank):
-    pass
+  '''
+  input: two parameters word and letter_bank. word represents an input from the user
+  letter_bank represents random letters drawn at the start of the game
+  output: returns True if user used letters to make the word from letter_bank; False if they did not
+  '''
 
-def score_word(word):
-    pass
+  word = word.upper()
```

Comment:
Hmmm, in this line, you define the local variable `word` and the parameter is named `word`. I think this line is fine to keep, but I wonder if changing the param name to `input_word` so this reads as

```python
def score_word(input_word)
  word = input_word.upper()
```
?